### PR TITLE
Run README through Grammarly, rename LICENSE to MIT-LICENSE

### PR DESCRIPTION
**Pull request by mensfeld**
This PR:
- Cleans up README a bit using Grammarly
- Renames LICENSE to standard MIT-LICENSE
- Updates copyright reference to match the transfer similar to how librdkafka was moved.

This is the beginning of my "daily" work on improving quality of documentation and code documentation of this gem.

### CODE REVIEW

**Comment by pboling** [comment]
> Renames LICENSE to standard MIT-LICENSE

Why is this desirable?  I've seen it both ways, and have waffled between both, but I've ended up going with the bundler default of `LICENSE.txt` for a few years now.  I'd like to understand why there might be an alternative preference.

File: README.md [hunk 1]
```diff
@@ -12,14 +12,14 @@
 The `rdkafka` gem is a modern Kafka client library for Ruby based on
 [librdkafka](https://github.com/edenhill/librdkafka/).
 It wraps the production-ready C client using the [ffi](https://github.com/ffi/ffi)
-gem and targets Kafka 1.0+ and Ruby versions that are under security or
-active maintenance. We remove Ruby version from our CI builds if they 
+gem and targets Kafka 1.0+ and Ruby versions under security or
+active maintenance. We remove the Ruby version from our CI builds if they
```
Comment (pboling) [inline README.md:16]:
```suggestion
active maintenance. We remove a Ruby version from our CI builds when they 
```

File: MIT-LICENSE [hunk 2]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) 2017 Thijs Cadier
+Copyright (c) 2017-2023 Thijs Cadier
+              2013, Maciej Mensfeld
```
[the remark this answers is not in the document]
Comment (mensfeld) [reply MIT-LICENSE:4]: not only that but also 2023 and not 2013

**Comment by mensfeld** [comment]
@pboling 

> > Renames LICENSE to standard MIT-LICENSE
> 
> Why is this desirable? I've seen it both ways, and have waffled between both in my own projects, but I've ended up going with the bundler default of `LICENSE.txt` for a few years now. I'd like to understand why there might be an alternative preference.

there is no one setting but I wanted to align it with the rest of the Karafka gems to have one convention where all MIT projects (all except Karafka as far as I remember) have MIT-LICENSE. It will help me later to write a set of cross-repo linters for quality around "meta" work. I hope this is not a problem.

**Comment by pboling** [comment]
No problem, the reasoning is sound.  I just want to know best practices wherever I can, and I hadn't seen much around LICENSE file naming.

**Comment by mensfeld** [comment]
>  I just want to know best practices wherever I can, and I hadn't seen much around LICENSE file naming.

Not really any standard and what I do is to use same convention across projects.

**Review by nijikon — APPROVED** [review]

**Pull request merged**